app/pb_kits/playbook/pb_progress_pills/_progress_pills.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= content_tag(:div,
+    id: object.id,
+    data: object.data,
+    class: object.classname) do %>
+    <div class="d-flex flex-column">
+      <div class="status d-flex">
+      <% if object.status != nil %>
+        <%= pb_rails("title", props: { text: object.status, tag: "h4", size: 4, dark: object.dark})%>
+        <%= pb_rails("body", props: { text: object.text, color: "light", dark:object.dark}) %>
+      <% end %>
+      </div>
+      <div class="pills">
+        <% 1.upto(object.steps) do |step| %>
+          <span class="pb_progress_pill<%= step <= object.active ? "_active" : "_inactive" %>"></span>
```

Comment:
Could this conditional be moved into the view object? Moving the conditional into the view object would make this view simpler, and it would be easier to test in the view object.